dev-util/xmake/patches/xmake-2.8.5.patchset

```diff
@@ -0,0 +1,54 @@
+From 44c5ae1ccefab0db49475b4f50f0fb81188eae9c Mon Sep 17 00:00:00 2001
+From: Christof Meerwald <cmeerw@cmeerw.org>
+Date: Fri, 15 Dec 2023 14:56:01 +0000
+Subject: [PATCH] adjust fallback for Haiku OS
+
+---
+ core/src/xmake/engine.c | 4 ++++
+ 1 file changed, 4 insertions(+)
+
+diff --git a/core/src/xmake/engine.c b/core/src/xmake/engine.c
+index f1b0b381b..e1d7a92e7 100644
+--- a/core/src/xmake/engine.c
++++ b/core/src/xmake/engine.c
+@@ -682,8 +682,12 @@ static tb_size_t xm_engine_get_program_file(xm_engine_t* engine, tb_char_t* path
+         static tb_char_t const* s_paths[] =
+         {
+             "~/.local/bin/xmake",
++#if defined(TB_CONFIG_OS_HAIKU)
++            "/boot/system/bin/xmake"
++#else
```

Comment:
As long as you can specify multiple paths to check then you should add `/boot/home/config/bin/xmake`.  Haiku packages can also be installed to the user home directory at `~/config/packages/` which changes the paths. Once that is added I think the recipe is probably ready to go.

Reply:
added `~/config/bin/xmake` and tested that it works for both cases